### DISCUSSION TITLE
Student Developer Pack needed urgently

### DISCUSSION THREAD

**Discussion by gboliveira011** [community/community#136489] [GitHub Education]
### Hi everyone,

Hi! I have just been approved for the GitHub education benefits, and I need to have the benefits for an assignment this semester by tonight.

**Comment by davevad93** [comment]
Hi @gboliveira011 ,

As stated in the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) "***If you haven't received your education benefits within 5 days of your approval status please raise a discussion on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education) and the Education community team will be able to assist in resolving this for you"***. 

If 5 days haven't passed yet, you have to wait in order to receive assistance from the Education team. In general, it takes a few days to receive benefits, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests;

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @gboliveira011 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
